src/data/events.json

```diff
@@ -163,5 +163,16 @@
     "location": "Chennai",
     "communityName": "Elastic Chennai User Group",
     "communityLogo": "https://ibb.co/RJphpSr"
+  },
+  {
+    "eventName": "Microsoft Developer Day",
+    "eventDescription": "Join the TechNexus Community for an exciting day of discussions, learning, and networking at Microsoft Developer Day! This event will bring together Microsoft MVPs and industry experts to share insights, trends, and experiences in the ever-evolving world of AI and software development. Discover the latest advancements, learn from real-world implementations, and connect with fellow developers who are shaping the future of technology. ### 📢 Final Agenda & Speaker Lineup: TBD (The final agenda will be announced in the first week of April.) Venue has limited seats! We’re looking forward to an insightful event that fosters learning and collaboration. Stay tuned for the agenda and speaker details!",
+    "eventDate": "2025-04-26",
+    "eventTime": "10:00",
+    "eventVenue": "TBD",
```

Comment:
Hey @BharathKumarMurugan Thanks a lot for the PR, but can we know when this is going to be changed and if you would be pulling a new PR when it changes. 

We'll merge it for now and you can update it once you get the venue confirmed. 

Cheers :)